content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -33,17 +33,17 @@ installing deb or rpm packages. The responsible SIG for kubeadm,
 but you may also build them from source for other OSes.
 
 
-### kubeadm Maturity
+### kubeadm maturity
 
 | Area                      | Maturity Level |
 |---------------------------|--------------- |
 | Command line UX           | GA             |
 | Implementation            | GA             |
-| Config file API           | beta           |
+| Config file API           | Beta           |
 | CoreDNS                   | GA             |
-| kubeadm alpha subcommands | alpha          |
-| High availability         | alpha          |
-| DynamicKubeletConfig      | alpha          |
+| kubeadm alpha subcommands | Alpha          |
+| High availability         | Beta           |
+| DynamicKubeletConfig      | Alpha          |
```

Comment:
Not sure, but if I got this right from Tim, we are not considering phases GA...

Reply:
Automatic Certificate Copy: Beta

Reply:
> Not sure, but if I got this right from Tim, we are not considering phases GA...

if not GA do we have a State for them?

> Automatic Certificate Copy: Beta

isn't it a core functionality of HA going to Beta?

Reply:
@neolit123 
Ok, agreed with your comments (no changes required)